src/App/Playbook/content/Showroom/index.js

```diff
@@ -0,0 +1,530 @@
+import React from "react";
+import { DocContainer } from "@docutils";
+
+const basename = process.env.basename;
+
+const Showroom = () => (
+	<section>
+		<div
+			className="container px-sm-0 playbook-mozaic"
+			style={{ radius: "2px" }}
+		>
+			<div className="row">
+				<div className="col-sm">
+					<video
+						playsInline
+						autoPlay
+						muted
+						loop
+						style={{ aspectRatio: "16 / 9" }}
+						width={710}
+						height={399}
+						poster={`${basename}img/documentation/playbook/playbook-showroom-1.avif`}
+					>
+						<source
+							src={`${basename}img/documentation/playbook/playbook-showroom-1-av1.mp4`}
+							type="video/mp4; codecs=av01.0.05M.08,opus"
+						/>
+						<source
+							src={`${basename}img/documentation/playbook/playbook-showroom-1.mp4`}
+							type="video/mp4"
+						/>
+					</video>
+				</div>
+			</div>
+			<div className="row mt-3">
+				<div className="col-sm">
+					<picture>
+						{/* <source
+							type="image/avif"
+							media="(min-width: 576px)"
+							srcSet={`${basename}img/documentation/playbook/playbook-showroom-2.avif`}
+						/> */}
```

Comment:
Reasoning:
"Why keeping code commented-out?" You want to ask ?
- Well, great question !! (I first added the images in avif too, to get better web-perf. But then designers asked me to yet increase the quality of the images. My issue is that I think the quality is good enough (3x everywhere) and I converted to avif in as high quality as possible and lossless. And with all the img/media content on this page the computers and phones are struggling to process and paint it all fluidly (well, they really struggle on a non 3000$ machine such as the crappy old laptop I am using, and several videos cannot even play). I do suspect this decision will change in the near future in an iteration for the necessity of actually having the page displaying its content fluidely even if it means slightly less quality). So here it is, ready to be un-commented at the snap of a finger 😉🤫